builtin/providers/aws/resource_aws_launch_configuration.go

```diff
@@ -206,3 +461,9 @@ func resourceAwsLaunchConfigurationDelete(d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func readBlockDevicesFromLaunchConfiguration(d *schema.ResourceData, launchConfiguration *autoscaling.LaunchConfiguration, autoscalingconn *autoscaling.AutoScaling) map[string]interface{} {
+	var blockDevices = make(map[string]interface{})
+	// Need to figure out how to determine the various instance types.
```

Comment:
@phinze, I mimic'd the write functionality of the EC2 Block instances. How do I determine whether a device is root, ephemeral, or ebs on the read?